_inc/client/setup-wizard/feature-toggle/map-feature-to-props.js

```diff
@@ -0,0 +1,1028 @@
+				upgradeLink,
+				info,
+			};
+		},
+	},
+
+	search: {
+		mapStateToProps: state => {
+			let upgradeLink;
+			let optionsLink;
+			const sitePlan = getSitePlan( state );
+			if (
+				'is-business-plan' !== getPlanClass( sitePlan.product_slug ) &&
+				! hasActiveSearchPurchase( state )
+			) {
+				upgradeLink = '#/plans';
+			} else {
+				optionsLink = '/wp-admin/admin.php?page=jetpack#/settings?term=search';
+			}
+
+			return {
+				title: __( 'Search' ),
+				details: __(
+					'Incredibly powerful and customizable, Jetpack Search helps your visitors instantly find the right content – right when they need it.'
+				),
+				checked: getSetting( state, 'search' ),
+				isPaid: true,
+				upgradeLink,
+				optionsLink,
+			};
+		},
+		mapDispatchToProps: dispatch => {
+			return {
+				onToggleChange: currentCheckedValue => {
+					return dispatch( updateSettings( { search: ! currentCheckedValue } ) );
+				},
+			};
+		},
+	},
+
+	sso: {
+		mapStateToProps: state => {
+			return {
+				title: __( 'Secure Sign On' ),
+				details: __( 'Add an extra layer of security.' ),
+				checked: getSetting( state, 'sso' ),
+				optionsLink: '#/settings?term=secure%20sign%20on',
+			};
+		},
+		mapDispatchToProps: dispatch => {
+			return {
+				onToggleChange: currentCheckedValue => {
+					return dispatch( updateSettings( { sso: ! currentCheckedValue } ) );
+				},
+			};
+		},
+	},
+
+	seo: {
+		mapStateToProps: state => {
+			return {
+				title: __( 'SEO' ),
+				details: __( 'Take control of the way search engines represent your site.' ),
+				checked: getSetting( state, 'seo-tools' ),
+				configureLink: '#/settings?term=seo',
+			};
+		},
+		mapDispatchToProps: dispatch => {
+			return {
+				onToggleChange: currentCheckedValue => {
+					return dispatch( updateSettings( { 'seo-tools': ! currentCheckedValue } ) );
+				},
+			};
+		},
+	},
+
+	sitemaps: {
+		mapStateToProps: state => {
+			return {
+				title: __( 'Sitemaps' ),
+				details: __( 'Automatically generate sitemaps for all your content.' ),
+				checked: getSetting( state, 'sitemaps' ),
+				optionsLink: '#/settings?term=sitemaps',
+			};
+		},
+		mapDispatchToProps: dispatch => {
+			return {
+				onToggleChange: currentCheckedValue => {
+					return dispatch( updateSettings( { sitemaps: ! currentCheckedValue } ) );
+				},
+			};
+		},
+	},
+
+	sharing: {
+		mapStateToProps: state => {
+			return {
+				title: __( 'Sharing' ),
+				details: __( 'Increase sharing of your posts and pages. ' ),
+				checked: getSetting( state, 'sharedaddy' ),
+				optionsLink: '#/settings?term=sharedaddy',
+			};
+		},
+		mapDispatchToProps: dispatch => {
+			return {
+				onToggleChange: currentCheckedValue => {
+					return dispatch( updateSettings( { sharedaddy: ! currentCheckedValue } ) );
+				},
+			};
+		},
+	},
+
+	shortcodes: {
+		mapStateToProps: state => {
+			return {
+				title: __( 'Shortcode Embeds' ),
+				details: __( 'Embed YouTube videos, and other content easily.' ),
```

Comment:
I don't know if we should mention Youtube here, since that's something you get with stock WordPress. Shouldn't we focus on embeds that Jetpack provides on top of WordPress?

Reply:
@jessefriedman thoughts on this?